midp/gfx.js

```diff
@@ -278,18 +273,40 @@ var currentlyFocusedTextEditor;
         converterFunc(rgbData, abgrData, width, height, 0, width);
 
         context.putImageData(ctxImageData, 0, 0);
-
-        setImageData(imageData, width, height, context);
     };
 
+    Native["javax/microedition/lcdui/ImageData.finalize.()V"] = function() {
```

Comment:
The finalize method is, in the standard JVM, called by the garbage collector before releasing an object. In our VM, we're never calling the finalize methods.